src/classes/tool_handlers.py

```
@@ -6332,6 +6332,1095 @@ def build_editor_snapshot_for_chat(max_chars: int = 5500) -> str:
+        for clip in clips:
+            reader = clip.data.get("reader", {})
+            source_path = reader.get("path") or clip.data.get("image", "")
+            if not source_path or not os.path.isfile(source_path):
+                results.append(f"Skipped clip {clip.id}: source file not accessible.")
+                continue
+
+            clip_start = float(clip.data.get("start", 0.0) or 0.0)
+            clip_end = float(clip.data.get("end", 0.0) or 0.0)
+            clip_duration = clip_end - clip_start
+
+            wav_path = tempfile.mktemp(suffix=".wav")
```

Ruff (0.16.3)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)